### ISSUE TITLE
[NEW TOOL] HAR File Sanitizer

### ISSUE THREAD

**Issue by justinlosh** [CorentinTh/it-tools#811]
**What tool do you want?**
Example: HAR File Sanitizer

**Describe the solution you'd like**
A tool similar to the Cloudflare or Google HAR Sanitizer (https://har-sanitizer.pages.dev/)

**Is their example of this tool in the wild?**
https://github.com/cloudflare/har-sanitizer

**Comment by miro444** [comment]
Yep, sanitization would be super useful after we saw what can happen with unsanitized HAR in https://www.rezonate.io/blog/har-files-attack-okta-customers/. Also analyzer to check the HAR stats.

**Comment by sharevb** [comment]
Hi @justinlosh and @miro444, implemented in #1531 and in my fork : https://sharevb-it-tools.vercel.app/har-sanitizer
It is adapted from Cloudflare HAR sanitizer here : https://github.com/cloudflare/har-sanitizer

And if you are interested in an up to date version of it-tools, with many improvements, new tools, and bug fixes, as this repo is almost no more maintained, I made a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)
